doc/installation.rst

```diff
@@ -257,54 +314,76 @@ You'll first want to create and activate a virtual environment in which to insta
 
 With ``virtualenvwrapper``:
 
-.. code-block:: bash
+.. code-block:: shell
 
    mkvirtualenv spyder-env
    workon spyder-env
 
 Otherwise, on macOS/Linux/Unix:
 
-.. code-block:: bash
+.. code-block:: shell
 
    python3 -m venv spyder-env
    source spyder-env/bin/activate
 
 or on Windows:
 
-.. code-block:: bash
+.. code-block:: batch
 
    python -m venv spyder-env
-   spyder-env/Scripts/activate.bat
+   spyder-env\Scripts\activate.bat
 
-After activating your environment, to install Spyder and its other dependencies, run ``pip install spyder``.
+After activating your environment, to install Spyder and its optional dependencies, run:
+
+.. code-block:: shell
+
+   pip install spyder numpy scipy pandas matplotlib sympy cython
+
+Or for a minimal installation, run:
+
+.. code-block:: shell
+
+   pip install spyder
 
 .. image:: /images/installation/installation-pip-install.gif
    :alt: Running Spyder installation with pip
 
-You may need to install a Qt binding (PyQt5) separately with ``pip`` if running under Python 2.
+To launch Spyder after installing, ensure your environment is activated and run the ``spyder`` or ``spyder3`` command.
```

Comment:
```suggestion
To launch Spyder after installing it, ensure your environment is activated and run the ``spyder`` or ``spyder3`` command.
```

Seems missing.

Reply:
You _can_ get away with eliding it, but better to have it I guess